recipes/movingpandas/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set version = "0.1.dev2" %}
+
+package:
+  name: movingpandas
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/m/movingpandas/movingpandas-{{ version }}.tar.gz
+  sha256: 5623a30942242336121a2394e4279861f7c7cf5643aca26fac5bccda49db6461
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - matplotlib
```

Please use `matplotlib-base` if your package doesn't explicitly need the Qt backend.
```suggestion
    - matplotlib-base
```

---

@xhochy Yes, thanks!